package.json

```diff
@@ -41,7 +41,7 @@
         "standard.enable": {
           "scope": "resource",
           "type": "boolean",
-          "default": true,
+          "default": false,
```

Comment:
Hmm, won't this turn off standard linting for anyone currently using the extension, when upgrading to this version? 🤔

Reply:
Yes, that's why it is a **BREAKING CHANGE**.

I don't think it is a great idea to enable `standard` linting by default globally, it should be enabled by each project individually so if one project use `eslint` instead of `standard` engines, it doesn't conflict.

Maybe we can do better ?
We can detect if one of the engines (`standard`, `semistandard`, `standardx` or `ts-standard`) is installed in `devDependencies` if so we enable the extension?
That way we don't need any settings in`.vscode/settings.json`, it just works out of the box (as one of `standard` goals).
What do you think?

Reply:
I think there should be three levels:

1. Globally enabled
2. Globally disabled
3. Globally enabled in projects that has a recognizable  `standard` setup

And the default should be the third option in my opinion.

Reply:
I agree too the default should be the third option, so we don't need any settings to use this extension by default. :+1:

Currently, the extension is working by default with the first option (Globally enabled), this PR was trying to make it work like the second option (Globally disabled).

When I will have some time, I will update the PR to make it work like the third option.
Thanks for making it clearer! :smile:

Reply:
Done! I updated the PR.

Also, I added in `CHANGELOG.md` what changed and bump the version number to `2.0.0`, so we can release a new version after this PR has been merged.